app/src/main/java/com/github/sdp_begreen/begreen/fragments/CameraFragment.kt

```diff
@@ -77,6 +83,22 @@ class CameraFragment : Fragment() {
         }
     }
 
+    /**
+     * Helper function to set up the search bar
+     */
+    private suspend fun setUpSearchBar(view: View) {
+
+        val users = db.getAllUsers()
+        val adapter = ArrayAdapter(requireContext(), android.R.layout.select_dialog_item, users)
+
+        //Getting the instance of AutoCompleteTextView
+        val actv = view.findViewById<AutoCompleteTextView>(R.id.userSearch)
```

Comment:
I understood that it is `AutoCompoeteTextView`, but I had to think about it to find. Maybe we could find a more explicit name. Maybe `autoCompleteTV` ? Let me know what you think.